plugins/woocommerce/src/Internal/ProductFilters/QueryClauses.php

```
@@ -239,13 +304,118 @@ public function add_attribute_clauses( $args, $chosen_attributes ) {
+	private function get_current_attribute_terms( array $all_terms, string $taxonomy, int $taxonomy_count ): array {
+		if ( 1 === $taxonomy_count ) {
+			return $all_terms;
+		}
+
+		return array_filter(
+			$all_terms,
+			function ( $term ) use ( $taxonomy ) {
+				return $term->taxonomy === $taxonomy;
+			}
+		);
+	}
```

PHPMD (2.15.0)

Avoid unused private methods such as 'get_current_attribute_terms'. (Unused Code Rules)

(UnusedPrivateMethod)